test/end-to-end/HACKING.md

```diff
@@ -0,0 +1,90 @@
+# Hacking on end to end test
+
+Here's where to get the code:
+
+    $ git clone https://github.com/weldr/cockpit-composer.git
+    $ cd test/end-to-end
+
+The remainder of the commands assume you're in the test/end-to-end folder.
+
+## Getting the development dependencies
+
+Cockpit-composer end to end test uses Node.js during case development.
+To make changes on Cockpit you'll want to install Node.js, NPM and
+various development dependencies.
+
+On Fedora:
+
+    $ sudo dnf install nodejs
```

Comment:
Shouldn't that be "npm"? That pulls in nodejs, but nodejs itself doesn't have /bin/npm.